plugins/org.yakindu.sct.generator.c/src/org/yakindu/sct/generator/c/StatemachineHeader.xtend

```diff
@@ -212,6 +215,11 @@ class StatemachineHeader implements IContentTemplate {
 			/*! Define dimension of the state configuration vector for history states. */
 		#define «type.toUpperCase»_MAX_HISTORY_STATES «historyVector.size»«ENDIF»
 		
+		«IF timed»
+		/*! Define maximum number of time events that can be active at once */
+		#define «type.toUpperCase»_MAX_ACTIVE_TIME_EVENTS «(it.sourceElement as Statechart).maxNumberOfParallelTimeEvents»
```

Comment:
- use naming extension as in c++ generator for calculating the constant name
- apply a consistent naming: 'ACTIVE' here vs 'parallel' everywhere else